### ISSUE TITLE
resource not found

### ISSUE THREAD

**Issue by oviliz** [cloudflare/worker-speedtest-template#3]
After publishing, when accessing https://speedtest.oviliz.workers.dev I'm given "resource not found". What I'm doing wrong?
This is my sequence:
```
wrangler generate speedtest https://github.com/cloudflare/worker-speedtest-template
cd speedtest/
nano wrangler.toml #adding the account_id
wrangler publish
```
Had the same problem when doing "wrangler preview".

**Comment by vasturiano** [comment]
Thanks for reaching out @oviliz!

So that we troubleshoot your issue further, could you please fill in the details from this [template](https://github.com/cloudflare/wrangler/blob/master/.github/ISSUE_TEMPLATE/bug_report.md).

Please also provide the content of your `wrangler toml` file and the output after running `wrangler preview` and/or `wrangler publish`.

In addition, it would also give us more info if you can run your `wrangler` command with `RUST_LOG=info` in front of it, and paste the output of that.

**Comment by trevyn** [comment]
Yeah, I'm not sure what the front-end to this is supposed to be, because only `/up` and `/down` routes seem to be exposed.

But you can still just do something like
```
curl "https://speedtest.your-id.workers.dev/down?bytes=10000000" > /dev/null
```

**Comment by danielro** [comment]
Has anyone figured out how to use this?
I am looking to use it as an API for a speed test.

**Comment by Baizhixi** [comment]
I had the same problem

**Comment by oviliz** [comment]
Sorry guys, I've personally given up.

**Comment by nataliescottdavidson** [comment]
### Add test card to workers endpoint



#### Environment

* operating system: MacOS Catalina 10.15.4
* output of `rustc -V`: `rustc 1.42.0 (b8cedc004 2020-03-09)`
* output of `node -v`: `v11.9.0`
* output of `wrangler -V`: `👷 ✨  wrangler 1.9.2`
* contents of `wrangler.toml`:
```
name = "slack-bot"
type = "webpack"
account_id = "04e8ec7979d1d07c39925ab7332a116e"
workers_dev = true
route = ""
zone_id = ""

```


#### Steps to reproduce

I was following the [workers tutorial](https://developers.cloudflare.com/workers/tutorials/build-an-application). After publishing I navigated to
my endpoint https://slack-bot.asylum.workers.dev/ and got the blank screen reading `resource not found`.

The app did work and publish github issue to my slack. 

I propose to add a [test card](https://en.wikipedia.org/wiki/Test_card) because other users will navigate to
the workers endpoint in attempt to debug. They should see something which confirms success.
